### ISSUE TITLE
Release 2.13

### ISSUE THREAD

**Issue by phillipberndt** [phillipberndt/pqiv#226]
Code currently builds cleany. I explored adding GTK 4 support, but the X11 related changes are extensive and I would probably break some old workarounds, so I won't go for that until there's demand.

As usual, I'll wait for a while to see if any bug reports tickle in. If there's none, I'll release the current code as 2.13.

**Comment by NHOrus** [comment]
I have a problem:
![image](https://github.com/phillipberndt/pqiv/assets/5390835/4bc165ad-5712-4c40-9bbc-9c3ab21b5f17)

**Comment by phillipberndt** [comment]
To clarify, is the font size of the emoji the problem?

**Comment by NHOrus** [comment]
Yes. Emoji grows beyond the size of the font. I'm not sure if it's bug in my fonts configuration. Bit it's a problem.

Отправлено из Yahoo Почты для Android 
 
  вт, 9 янв. 2024 в 10:41 Phillip ***@***.***> написал(-а):   


To clarify, is the font size of the emoji the problem?

**Comment by NHOrus** [comment]
I would love proper release very much, with my issue going to 2.13.1 or something.

**Comment by phillipberndt** [comment]
Forgot to leave an update: There's some issues with rendering in Wayland that I would like to address before issuing a release.

**Comment by phillipberndt** [comment]
I worked around those by just preferring the x11 backend on the GDK side. Looking into whether the emoji issue has a similarly simple workaround now.

**Comment by phillipberndt** [comment]
Unfortunately, not trivial to reproduce. For me, the 🐈 renders at the same size as the rest of the line for me.

Which version of Pango do you run? I'm testing with 1.50.6+ds-2 on an Ubuntu 20.04 running Gnome, without any modifications.

**Comment by NHOrus** [comment]
This problem was caused by awful mess of my font configuration.
After turning said awful mess off, problem has gone away. I don't think I can reproduce it myself.
Pango 1.51.0 on gentoo.

**Comment by phillipberndt** [comment]
Wayland support is fixed now, I finally had the nerves to work my way through the GTK CSD implementation. I'll give it a few days to see if anyone reports any issues.

**Comment by phillipberndt** [comment]
Release pushed: https://github.com/phillipberndt/pqiv/releases/tag/2.13

Keeping this open for a while to track issues if there's any.
